src/Stryker.Core/Stryker.Core/MutationTest/MutationTestProcess.cs

```diff
@@ -174,21 +181,17 @@ public StrykerRunResult Test(StrykerOptions options)
         public void Optimize(TestCoverageInfos coveredMutants)
         {
             if (coveredMutants == null)
-            {
-                return;
-            }
-            var covered = new HashSet<int>(coveredMutants.CoveredMutants);
-            if (covered.Count == 0)
             {
                 Logger.LogDebug("No mutant is covered by any test, no optimization done.");
                 return;
             }
+            var covered = new HashSet<int>(coveredMutants.CoveredMutants);
+
             Logger.LogDebug("Optimize test runs according to coverage info.");
-            var report = new StringBuilder();
-            var nonTested = Input.ProjectInfo.ProjectContents.Mutants.Where(x =>
-                x.ResultStatus == MutantStatus.NotRun && !covered.Contains(x.Id)).ToList();
-            const MutantStatus mutantResultStatus = MutantStatus.Survived;
-            foreach (var mutant in nonTested)
+
+            var nonCoveredMutants = Input.ProjectInfo.ProjectContents.Mutants.Where(x => x.ResultStatus == MutantStatus.NotRun && !covered.Contains(x.Id)).ToList();
+            const MutantStatus mutantResultStatus = MutantStatus.NoCoverage;
```

Comment:
I don't think it's neccesary to const declare an enum value especially when the variable name is less descriptive than the enum name

Reply:
@richardwerkman The const is also used on line 210